README.md

```diff
@@ -139,6 +139,12 @@ yarn workspace <workspace-name> <workspace-command>
  └── yarn.lock
 ```
 
-<!-- ## Deploy -->
+## Deploy
 
-<!-- TODO -->
+[NextJS](https://nextjs.org/) apps can be easily deployed to custom Node.js servers, as Docker images, static HTML exports, and [Vercel](https://vercel.com/) (recommended).
```

Comment:
This text is not relevant because it speaks about NextJS whereas we have a monorepo project?